### ISSUE TITLE
Add epicmenu option to automorph (PR #4715)

### ISSUE THREAD

**Issue by sprunk** [ZeroK-RTS/Zero-K#4806]
Apparently people do make ctrl+shift blocks of shields after all (and mind the morph despite this).

**Comment by Licho1** [comment]
Whats automorph in this context?

**Comment by sprunk** [comment]
If you build a morphable building (geo, cloaker or shield) when holding Ctrl, it will morph as soon as it is ready. This is because you often want to build cloaker or shield just to get a mobile version, or upgrade geo asap.

The problem is that Shift+Ctrl is also used to drag rectangles of buildings, so now when you make (for example) 15 shields in a 3x5 rectangle they will each start morphing as they finish because the combination involves Ctrl. I originally thought this conflict is not a problem, because shield and cloaker buildings are never _viable_ to make in a rectangle and geo can only be placed on single spots, so I didn't request a toggle with the PR that added the feature. But some player did make a grid of shields after all in a co-op game and complained. So this should have a settings option to turn off.

**Comment by GoogleFrog** [comment]
Having a setting to turn it off isn't going to stop the player complaining. Perhaps Ctrl should only automorph if a single command is issued.

**Comment by sprunk** [comment]
> Having a setting to turn it off isn't going to stop the player complaining.

That sounds fine as long as they can be directed towards the setting upon a complaint.

> Perhaps Ctrl should only automorph if a single command is issued.

So essentially change the hotkey from [Ctrl] to [Ctrl and NOT Shift]? Sounds good.

**Comment by GoogleFrog** [comment]
Yes. Or really, just click a bunch of times when placing. The whole point of Ctrl is to give orders at a single time, then come back to find morphed units waiting.
